### DISCUSSION TITLE
I get this bellow error

### DISCUSSION THREAD

**Discussion by hemanta009** [ImageMagick/ImageMagick#5686] [Help]
FailedToExecuteCommand `"gs" -sstdout=%stderr -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -dMaxBitmap=500000000 -dAlignToPixels=0 -dGridFitTT=2 "-sDEVICE=pngalpha" -dTextAlphaBits=4 -dGraphicsAlphaBits=4 "-r72x72" -dPrinted=false "-sOutputFile=C:/Windows/Temp/magick-HIQAHJJqj4GVYi8My1dw8gtijEk3nkv2%d" "-fC:/Windows/Temp/magick-evnDZMCmlE3_UQlxq0hJnrCo4eiIRugQ" "-fC:/Windows/Temp/magick-gxOuNXlEOHVWaujFJsM8Xxg20984oIdb"' (The system cannot find the file specified. ) @ error/delegate.c/ExternalDelegateCommand/516

**Comment by hemanta009** [comment]
My code in laravel
$image = new Imagick(public_path('uploads/ECG_Hemanta Kundu api.pdf'));
        $image->thumbnailImage(450, 450, true);
        $saveImagePath = public_path('converted.jpg');
        $image->writeImage($saveImagePath);

**Comment by fmw42** [comment]
Your version of Imagemagick must have Ghostscript installed to read PDF files. Is it installed in Imagemagick?

**Comment by hemanta009** [comment]
How to install Ghostscript in Imagemagick

**Comment by hemanta009** [comment]
How to install Ghostscript in Imagemagick in windows

**Comment by snibgo** [comment]
Install Ghostscript for Windows from https://ghostscript.com/releases/gsdnld.html

ImageMagick should then be able to find it.
